packages/odata-service-writer/src/updates.ts

```diff
@@ -43,13 +44,15 @@ export function updateManifest(basePath: string, service: OdataService, fs: Edit
 export function updatePackageJson(path: string, fs: Editor, addMockServer: boolean) {
     const packageJson = JSON.parse(fs.read(path));
     packageJson.devDependencies = packageJson.devDependencies ?? {};
-    packageJson.ui5 = packageJson.ui5 ?? {};
-    packageJson.ui5.dependencies = packageJson.ui5.dependencies ?? [];
+    if (!hasUI5CliV3(packageJson.devDependencies)) {
+        packageJson.ui5 = packageJson.ui5 ?? {};
+        packageJson.ui5.dependencies = packageJson.ui5.dependencies ?? [];
```

Comment:
Should we remove these sections if they are present and the version is 3+?

Reply:
Similar as discussion from here https://github.com/SAP/open-ux-tools/pull/935/files/69fd7b5348091cc5d60368c56709261a2f960a7c#r1122681360 so I went with the same approach here.